### PR TITLE
Client tabs, whitespace, Beautify, JSHint, JSCS. that -> _this

### PR DESCRIPTION
**Pull request by grempe**
No functional changes, just cleanup in the JS client code.

### CODE REVIEW

**Comment by daviddahl** [comment]
w00t! Sorry for the delay!

**Comment by grempe** [comment]
Thx.  Working on a few other things now so will likely pause on Crypton for
a bit.

File: client/src/account.js [hunk 1]
```diff
@@ -11,430 +11,454 @@
...
+        // need to update the database
+
+        var currentAccount = newSession.account;
+
+        // Replace all salts with new ones
+        var keypairSalt = crypton.randomBytes(32);
+        var keypairMacSalt = crypton.randomBytes(32);
+        var signKeyPrivateMacSalt = crypton.randomBytes(32);
+
+        var srp = new SRPClient(username, newPassphrase, 2048, 'sha-256');
```
Comment (daviddahl) [inline client/src/account.js]: So it turns out that Kloak is breaking inside of SRPClient constructor, What was changed here, this changeset just looks like a style/tabs/spaces change? Weird.

Comment (daviddahl) [inline client/src/account.js]: Actually it is happenieng wherever we create an SRPClient. The group ('2048' arg) is always undefined

Comment (grempe) [inline client/src/account.js]: Is it limited to the line number you commented on? Or elsewhere? What exactly are you seeing when it fails? The line you commented on looks the same.

**Pull request merged**